.github/workflows/close-issues.yml

```diff
@@ -0,0 +1,93 @@
+name: Close Unauthorized Issues
+
+on:
+  workflow_dispatch:
+  issues:
+    types: [opened]
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight, in case issues were missed due missing workflow runs
+
+jobs:
+  close-unauthorized-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      # XXX: This *could* be done in Bash by abusing GitHub's own tool to interact with its API
+      # but that's too much of a hack, and we'll be adding a layer of abstraction. github-script
+      # is a workflow that eases interaction with GitHub API in the workflow run context.
+      - name: "Close 'unauthorized' issues"
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const ALLOWED_USERS = ['vaxerki', 'fufexan', 'NotAShelf'];
+            const CLOSING_COMMENT = 'Users are no longer allowed to open issues themselves, please open a discussion instead.\n\nPlease see the [wiki](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) on why this is the case.\n\nWe are volunteers, and we need your cooperation to make the best software we can. Thank you for understanding! ❤️\n\n[Open a discussion here](https://github.com/hyprwm/Hyprland/discussions)';
+
+            async function processIssues() {
```

Comment:
is it not possible to get the ID of the issue that triggered the workflow here?

Reply:
I don't think so? Not sure if the GH API actually exposes the issue ID as a variable.

Reply:
I looked at the list of default env vars, but it doesn't look like it is exposed anywhere. We have to query the API either way. Pushed a fix to try and handle larger number of issues via pagination.

Reply:
oke